jbmc/src/java_bytecode/java_utils.cpp

```diff
@@ -464,3 +464,14 @@ symbolt &fresh_java_symbol(
   return get_fresh_aux_symbol(
     type, name_prefix, basename_prefix, source_location, ID_java, symbol_table);
 }
+
+optionalt<irep_idt> owning_class(const symbolt &symbol)
+{
+  const irep_idt &class_id = symbol.type.get(ID_C_class);
+  return class_id.empty() ? optionalt<irep_idt>{} : class_id;
```

Comment:
:pick: can `optionalt<irep_idt>{}` just be `{}`?

Reply:
Nope, because of the usage of the ternary. I'd have to refactor this into an `if` instead in order to be able to use just `{}`.